javascript-data-model/order-history.js

```diff
@@ -0,0 +1,63 @@
+var orderHistory = [
+  {
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    totalPrice: 34.00,
+    items: [{
```

Comment:
almost there! just make sure to add the rest of the item data to the item object for all orders, author, item-price